### PR TITLE
cli: support --privileged for exec

### PR DESCRIPTION
**Pull request by Ace-Tang**
Signed-off-by: Ace-Tang <aceapril@126.com>

### Ⅰ. Describe what this PR did

support cli `pouch exec --privileged`

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

no need

### CODE REVIEW

**Comment by rudyfly** [comment]
How to add test case to check it works?
I think you should merge with this pr: #2533

**Comment by Ace-Tang** [comment]
This is only a client part code, so I think you make it too complicated. We just add test in pr #2355 , this just two different type pr, no related

**Comment by Ace-Tang** [comment]
The test is failed, /cc @rudyfly 
```
FAIL: /home/travis/gopath/src/github.com/alibaba/pouch/test/cli_run_network_test.go:28: PouchRunNetworkSuite.TestRunWithPing
/home/travis/gopath/src/github.com/alibaba/pouch/test/cli_run_network_test.go:34:
    res.Assert(c, icmd.Success)
/home/travis/gopath/src/github.com/alibaba/pouch/vendor/github.com/gotestyourself/gotestyourself/icmd/command.go:61:
    t.Fatalf("at %s:%d - %s\n", filepath.Base(file), line, err.Error())
... Error: at cli_run_network_test.go:34 - 
Command:  /usr/local/bin/pouch run --name TestRunWithPing registry.hub.docker.com/library/busybox:1.28 ping -c 3 www.taobao.com
ExitCode: 1
Error:    exit status 1
Stdout:   PING www.taobao.com (66.102.255.57): 56 data bytes
--- www.taobao.com ping statistics ---
3 packets transmitted, 0 packets received, 100% packet loss
Stderr:   
Failures:
ExitCode was 1 expected 0
Expected no error
```

File: cli/exec.go [hunk 1]
```diff
@@ -73,7 +75,7 @@ func (e *ExecCommand) runExec(args []string) error {
 		AttachStderr: !e.Detach,
 		AttachStdout: !e.Detach,
 		AttachStdin:  !e.Detach && e.Interactive,
-		Privileged:   false,
+		Privileged:   e.Privileged,
```
Comment (fuweid) [inline cli/exec.go]: there is only client side change. And I didn't see that the daemon can use the field. 

I think you should add the `NoNewPrivileges` here.
https://github.com/alibaba/pouch/blob/8abf067257260a1570ed0a09bff2b16df9a1b899/daemon/mgr/container_exec.go#L88-L98

Comment (Ace-Tang) [reply cli/exec.go]: two different things, `NoNewPrivileges`  https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt , `Privileged` here related with container level

Comment (fuweid) [reply cli/exec.go]: anyway, LGTM. and wait for the CI.

**Comment by Ace-Tang** [comment]
closed by #2533

**Pull request closed without merging**